Formula/p/pgroonga.rb

```diff
@@ -24,10 +24,10 @@ class Pgroonga < Formula
 
   depends_on "pkg-config" => :build
   depends_on "groonga"
-  depends_on "postgresql@14"
+  depends_on "postgresql@16"
```

Comment:
May be a bit difficult to migrate this while keg-only. `postgresql@14` is linked and set to pick up `HOMEBREW_PREFIX/` paths so other formulae will just work. All other `postgresql@*` formulae won't work this way and may need some extra hacks.